src/NuGet.Services.EndToEnd/Support/Configuration/GalleryConfiguration.cs

```diff
@@ -12,7 +12,7 @@ public class GalleryConfiguration
     {
         public string GalleryBaseUrl { get; set; }
 
-        public AzureCloudServiceDetails ServiceDetails { get; set; }
+        public AzureCloudServiceOrSearchDetails ServiceDetails { get; set; }
```

Comment:
Would it be possible to do this change without affecting the gallery config?

Reply:
This config will be used by the gallery e2e tests as well, so this is the minimum change required to modify it. Its an additive change so I don't think it should break anything existing. I have the PR for the new configs https://nuget.visualstudio.com/_git/NuGetBuild/pullrequest/978?_a=files&path=%2FE2EConfig%2FProd-TestGalleryUSNCStaging.json shows the gallery config

Reply:
that is on the `SearchServiceConfiguration` object though, not `GalleryConfiguration`

Reply:
Oh you mean not rename the class?

Reply:
With the removed mentions of "azure search" and `UseConfiguredUrls`, maybe it's just `ServiceDetails` instead of `AzureCloudServiceDetails`. My resistance was having a type with the name "search" in it under `GalleryConfiguration`